### DISCUSSION TITLE
Prism template pack does not support visual studio 2022

### DISCUSSION THREAD

**Discussion by Yuexs** [PrismLibrary/Prism#2611] [General]
Prism template pack does not support visual studio 2022

**Comment by dansiegel** [comment]
You are correct it does not.

**Reply by mndarren** [reply]
Is it possible that Prism Template Pack will support VS 2022 in the future?

**Reply by brianlagunas** [reply]
Yes, it will one day

**Reply by mndarren** [reply]
Sounds great! Since we are using VS 2022 and Prism right now.  Thanks Brian!
